### PR TITLE
Update animals

### PR DESCRIPTION
**Pull request by taldr27**
In this milestone I:
- Added more animals to the table.
- Created new queries.
- Answer some questions with queries.
- Added License file.

# **Required screenshots:**

3-delete

# **Custom queries screenshots:**

# **Required changes screenshots:**
CHANGE1

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -16,3 +16,44 @@ SELECT * FROM animals WHERE neutered='t';
...
+/* */
+BEGIN;
+UPDATE animals SET species= 'digimon' where name LIKE '%mon';
+UPDATE animals SET species= 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT * FROM animals;
+/* */
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (Munsa1) [inline queries.sql:36]: - [x] Before a rollback kindly add a select statement to check that changes have been made

File: queries.sql [hunk 2]
```diff
@@ -16,3 +16,44 @@ SELECT * FROM animals WHERE neutered='t';
...
+/* New queries */
+BEGIN;
+UPDATE animals SET species= 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+/* */
+BEGIN;
+UPDATE animals SET species= 'digimon' where name LIKE '%mon';
+UPDATE animals SET species= 'pokemon' WHERE species IS null;
+COMMIT;
```
Comment (Munsa1) [inline queries.sql:30]: - [x] Again here kindly add a select statement to check that changes where made to the database before you commit

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @taldr27, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (rloterh) [reply queries.sql:30]: well resolved! ✔️

File: queries.sql (hunk 1 above)
Comment (rloterh) [reply queries.sql:36]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Diego,  

Your project is complete!🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Queries properly implemented

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by taldr27** [comment]
> ## Status: Approved ✔️
> Hi Diego,
> 
> Your project is complete!🍾 Congratulations! 🎉💯🌟
> 
> ## Highlights 👍
> * Queries properly implemented
> 
> Keep soaring higher! 💻 🍷 🚀
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you!

**Comment by taldr27** [comment]
@rloterh Thank you!

**Pull request merged**